src/test/java/ay2122s1_cs2103t_w16_2/btbb/model/order/UniqueOrderListTest.java

```diff
@@ -123,6 +126,48 @@ public void getTopTenOrderClients_listContainsOrdersForElevenClients_returnsTopT
         assertEquals(expectedTopTenClients, topTenClients);
     }
 
+    @Test
+    public void getTopTenOrderRecipes_listContainsOrdersForElevenRecipes_returnsTopTenRecipes() {
+        Order orderForAmy = new OrderBuilder(ORDER_FOR_AMY).withRecipeName(new GenericString("Latte"))
+                .withQuantity(new Quantity("1")).build();
+        Order orderForBob = new OrderBuilder(ORDER_FOR_BOB).withRecipeName(new GenericString("Chocolate Cake"))
+                .withQuantity(new Quantity("2")).build();
+        Order orderForImran = new OrderBuilder(ORDER_FOR_IMRAN).withRecipeName(new GenericString("Salad"))
+                .withQuantity(new Quantity("3")).build();
+        Order orderForHoon = new OrderBuilder(ORDER_FOR_HOON).withRecipeName(new GenericString("Aglio Olio"))
+                .withQuantity(new Quantity("4")).build();
+        Order orderForIda = new OrderBuilder(ORDER_FOR_IDA).withRecipeName(new GenericString("Porridge"))
+                .withQuantity(new Quantity("5")).build();
```

Comment:
Can you add new orders in typical orders instead? Afterwards, you can use a for loop to change the quantity of all orders and add it into the uniqueOrderList. (see getTopTenClients for reference)